example/index.js

```diff
@@ -96,15 +95,8 @@ app.all("/", function(req, res, next) {
     res.locals.title = "Register Transaction";
 
     // Build the notification URL.
-    var notificationUrl = url.parse(req.originalUrl);
-    notificationUrl.hostname = req.hostname;
-    notificationUrl.protocol = req.protocol;
-    notificationUrl.host = req.get("host");
-    notificationUrl.pathname = path.join(
-        notificationUrl.pathname,
-        "notification"
-    );
-    notificationUrl = url.format(notificationUrl);
+    var notificationUrl = new URL(path.join(req.originalUrl, 'notification'), `${req.protocol}://${req.get('host')}`);
```

Comment:
The expressions passed to URL are begging to be split onto separate lines, maybe even assigned to const/let/var for readability.